apis/Google.Cloud.Firestore/Google.Cloud.Firestore.Tests/QueryTest.cs

```diff
@@ -992,16 +992,20 @@ public void Equality_Where()
             EqualityTester.AssertEqual(query,
                 equal: new[] {
                     col.WhereEqualTo("field", 10), // Same value
+                    col.WhereEqualTo("field", 10L), // Same value after serialization
                     col.WhereEqualTo(new FieldPath("field"), 10), // FieldPath instead of Field
                 },
                 unequal: new[] {
                     query.WhereEqualTo("field", 10), // Repeated filter doesn't replace (at the moment, anyway)
                     col.WhereEqualTo("otherfield", 10), // Different field
                     col.WhereLessThan("field",  10), // Different operator
                     col.WhereEqualTo("field", 20), // Different value
+                    col,// Query with no filters
                 }
             );
 
+            var queryWithoutFilter = col.Select("a"); // query without any filter(query._filters = null).
```

Comment:
Any reason this can't just be added to the unequal list above?

Reply:
Here we want to test the equality when First Query(this) does not have a filter but 2nd query(other) have filters.

Reply:
Ok got it  Thanks for the explanation Jon